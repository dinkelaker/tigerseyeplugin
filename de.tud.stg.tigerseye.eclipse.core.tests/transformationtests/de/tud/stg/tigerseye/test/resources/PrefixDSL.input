start state 1

start
state 2

state 5

state

6

i = start
	