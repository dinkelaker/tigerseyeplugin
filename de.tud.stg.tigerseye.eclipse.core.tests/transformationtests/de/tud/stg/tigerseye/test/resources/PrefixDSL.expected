start__state__p0(1)

getStart()
state__p0(2)

state__p0(5)

state__p0(6)

i = getStart()
